tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/lifecycle/MutlipleMethodsBulkheadLifecycleService.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.fault.tolerance.tck.bulkhead.lifecycle;
+
+import org.eclipse.microprofile.faulttolerance.Bulkhead;
+
+import javax.enterprise.context.Dependent;
+
+@Dependent
+public class MutlipleMethodsBulkheadLifecycleService {
+    @Bulkhead(8)
+    public void service1(Barrier barrier) {
+        barrier.await();
+    }
+
+    @Bulkhead(8)
```

Comment:
It works better if you use a different number.

Reply:
The purpose is to verify that they are different (even though they perhaps have the same number of allowed executions).